introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/Amazon_JumpStart_Sentence_Pair_Classification.ipynb

```diff
@@ -41,8 +41,9 @@
     "4. [Finetune the pre-trained model on a custom dataset](#4.-Finetune-the-pre-trained-model-on-a-custom-dataset)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        metric_definitions = metric_definitions_per_model[model_id[0:10]]
</pre><p>What is being parsed with this hard-coded indexing? I see it differs slightly among the notebooks - <code>[0:7], [0:9], [0:10]</code> - how do customers know which indices to use if they revise this code? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3334/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
updated the code to search in a uniform way in all Notebooks.